ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -736,6 +739,20 @@ open class DrawerController: UIViewController {
                 offset = min(offset, 0)
             }
         }
+
+        // This resizingBehavior(expand) avoid the user to dismiss the drawer
+        if resizingBehavior == .expand && presentationDirection == .up {
```

Comment:
What about the scenario where the presentation direction is .down?

Reply:
My bad, I will handle it. 

Actually tried the scenario with down direction. In that case, offsetWithResistance is going beyond 30 (maximum offset to dismiss). So I am thinking to make the offset as 0 in case of down direction like handled in case of expand. Thoughts?

Reply:
Why is the offsetWithResistance is going beyond 30?
I'm not sure I understand what you meant by "make the offset as 0 in case of down direction like handled in case of expand"...

Reply:
Because in case of .down direction, offset is going beyond 300. I debugged in case .dismiss behavior as well. We are not using offsetWithResistance in case of .down direction.

Reply:
I think the offset going beyond the resizingThreshold is not the issue per se. We could write logic around that.
The problem is that the drawer is presented from a custom base and the resistance is currently just moving the entire drawer on top of the content underneath (instead of "compressing" the drawer to make it look like it was being dismissed into its custom base).

![resistance_custom_base](https://user-images.githubusercontent.com/68076145/92518318-fc7e9d00-f1cc-11ea-8ec4-94ab9ed51814.png)

We can treat that as a separate issue since it's not introduced by these changes.